src/tests/Microsoft.PowerFx.Core.Tests/CheckResultTests.cs

```diff
@@ -394,7 +394,7 @@ public void TestIR()
 
             var ir = check.ApplyIR();
             Assert.NotNull(ir);
-            Assert.Equal("BinaryOp(AddNumbers, Number(1), Number(2))", ir.TopNode.ToString());
+            Assert.Equal("AddNumbers:n(1:n, 2:n)", ir.TopNode.ToString());
```

Comment:
>"AddNumbers:n(1:n, 2:n) [](http://example.com/codeflow?start=25&length=23)

I'd prefer AddNumbers(1:n, 2:n):n
Feel free to disagree

Reply:
I hear you, it would overall be more consistent.  My concern is that for lots of nested calls in a complex epxression it will become difficult to match up the return type with the function. 

For example: `IR(Max(Max(Max(Today(),DateAdd(Today(),30)),Now()),Sqrt(2)))` would result in 
```
Max:n(DateTimeToNumber:n(Max:d(DateToDateTime:d(Max:D(Today:D(), DateAdd:D(Today:D(), 30:n))), Now:d())), Sqrt:n(2:n))
```

If the end type was after the closing paren, then it would be:
```
Max(DateTimeToNumber(Max(DateToDateTime(Max(Today():D, DateAdd:D(Today():D, 30:n):D):d):d, Now():d):n), Sqrt:n(2:n)):n
```
What's the return type of the second Max?  It's not as clear, one has to do a lot of paren matching.

So, I'd prefer with the function name for now.  We can try it this way and change it later if desired.